### PR TITLE
EVG-7692: Add "Spawn host" link to task metadata panel

### PR DESCRIPTION
**Pull request by SupaJoon**
https://jira.mongodb.org/browse/EVG-7692
Spawn host link is shown only when spawnHostLink is not null in the Task gql response[](url)
Screen Shot 2020-04-14 at 3 07 48 PM

### CODE REVIEW

File: cypress/integration/task_metadata_card.ts [hunk 1]
```diff
@@ -71,31 +71,51 @@ describe("Task Metadata Card", function() {
...
   if (get(xhr, resBodyPath)) {
-    dateContainer.should("not.be.empty");
-  } else {
-    dateContainer.should("be.empty");
+    dateContainer.should(doesExist);
+    return true;
   }
+  dateContainer.should(doesNotExist);
+  return false;
 };
```
Comment (tgrander) [inline cypress/integration/task_metadata_card.ts:121]: seems like something that could be useful to the rest of the tests. can this be generalized into a util?

Comment (SupaJoon) [reply cypress/integration/task_metadata_card.ts]: sure

**Comment by SupaJoon** [comment]
evergreen-retry

**Comment by SupaJoon** [comment]
evergreen retry

**Pull request merged**